features-json/css-sticky.json

```diff
@@ -35,6 +35,9 @@
     },
     {
       "description":"A parent with overflow set to `auto` will prevent `position: sticky` from working in Safari"
+    },
+    {
+      "description":"Browsers implement the dynamic change in the height of a sticky element [differently](https://stackoverflow.com/q/62249752/2932484).
```

Comment:
@Fyrd 
Hey there, I'd like to ask if this reversal wasn't by any chance by mistake: https://github.com/Fyrd/caniuse/commit/4de867c844353bef021334a8ccfdee9da81e460b#diff-f95bcebefec83d23b0ae8a0b7f35d507L40